Formula/chruby-fish.rb

```diff
@@ -12,9 +12,15 @@ class ChrubyFish < Formula
     sha256 "d56f48b0a49d0c381bffc28ffe320019b89a87a708ec51f24218a0911104bd78" => :mavericks
   end
 
+  depends_on "fish" => :recommended
   depends_on "chruby" => :recommended
 
   def install
     system "make", "install", "PREFIX=#{prefix}"
   end
+
+  test do
+    output = `fish -ic '. #{share}/chruby/chruby.fish; chruby --version'`
```

Comment:
Any documentation on `system_output`? I seem to be unable to find it in this repository, or the `brew` one.